server/models/User.js

```diff
@@ -0,0 +1,27 @@
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+
+const UserSchema = new Schema({
+	firstName: {
+		type: String,
+		required: true,
+		minlength: 2
+	},
+	lastName: {
+		type: String,
+		required: true,
+		minlength: 2
+	},
+	email: {
+		type: String,
+		required: true,
+		minlength: 5,
+		unique: true
+	},
+	pass: {
+		type: String,
+		required: true
+	}
+}, {collection: 'users'});
```

Comment:
This option isn't really necessary for our use case, since we're specifying the name below